src/odemis/util/img.py

```diff
@@ -466,9 +466,8 @@ def rescale_hq(data, shape):
     """
     # TODO: make it faster
 
-    out = numpy.empty(shape, dtype=data.dtype)
     scale = tuple(n / o for o, n in zip(data.shape, shape))
-    scipy.ndimage.interpolation.zoom(data, zoom=scale, output=out, order=1, prefilter=False)
```

Comment:
This seems to only work with 2D images (and also RGB images?). Very probably will not work with 5D images.
=> Based on the ndim, use `ndimage.zoom` or `misc.imresize`.
=> Check all the test cases in img_test pass.
=> Add an extra test case in the img_test to check rescale_hq() on an RGB image.

Also, change the scale of imresize() to a tuple, so that it works if the ratio is not kept (and that avoids computing the scale in the first place).